src/TreeHouse/BuckarooBundle/Request/AbstractTransactionRequest.php

```diff
@@ -45,6 +60,7 @@ public function toArray()
             'amount' => number_format($this->amount->getAmount() / 100, 2, '.', ''),
             'currency' => $this->amount->getCurrency()->getCode(),
             'invoicenumber' => $this->invoiceNumber,
+            'description' => (string) $this->description,
```

Comment:
Klopt maar dat heeft er niets mee te maken, de plek waar je nu kijkt gaat om serializen; dat moet toevallig altijd een string zijn, of hij nou leeg is of niet.